playbooks/check-versions.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: localhost
+  tasks:
+    - name: Fail if Ansible version is too old
+      fail:
+        msg: "Installed Ansible version is too old. Upgrade to at least 2.12.7"
+      when: (ansible_version.major < 2) or
+            (ansible_version.major == 2 and ansible_version.minor < 12) or
+            (ansible_version.major == 2 and ansible_version.minor == 12 and ansible_version.revision < 7)
```

Comment:
no need to check revision, 2.12.x is enough, so we don't bother user to update every time we bump patch version